components/pkg-export-kubernetes/src/manifest.rs

```diff
@@ -101,6 +101,8 @@ impl Manifest {
                           .map(str::to_string)
                           .unwrap_or_else(|| format!("{}-{}", pkg_ident.name, version_suffix));
 
+        let formatted_resource_name = Manifest::formatted_metadata_name(&name);
```

Comment:
I should've mentioned this in my initial review, but given that the replace is just a single line of code, you could do it here, and eliminate the separate function.  It would simplify things a bit.

Reply:
Good point, but I preferred to put formatted_metadata_name in a separate method so that I could test it. The "new_from_cli_matches" method which calls this method is extremely difficult to test (it currently has no test coverage on it - the existing tests cover the "generate" method, but not the "new_from _cli_matches" method. This exporter does need a much larger refactor to make it easier to test and to work in, but I chose the simpler, easier to test solution for now.

Reply:
In the future, I would certainly be open to doing a larger scale refactor, but with the list of priorities right now the simpler, slightly better solution seemed to be the best option.

Reply:
👍 for better testability!